src/ferry_planner/schedule.py

```diff
@@ -48,6 +53,17 @@ def __call__(
     ) -> FerrySchedule | None: ...
 
 
+class HtmlParseResult:
+    redirect_url: str | None = None
```

Comment:
```suggestion
    redirect_url: str = ""
```
A `None` value has no special meaning here, the null string `""` will work identically in `if not redirect_url:`/`if redirect_url:` conditions.